README.md

```diff
@@ -61,6 +61,17 @@ $ cargo install --locked --path cargo-near
 ```
 </details>
 
+## DISCLAIMER
+
+If you want a less stressful experience, and just want to run a regular once-used-to-be
+build, run 
+
+```bash
+# equivalent to behaviour of `cargo near build` of version < 0.7.0 
+cargo near build --no-docker --no-locked
+``` 
+instead of `cargo near build`.
+
```

Comment:
This sounds like we discourage `cargo near build` and even moved it upfront. This is not the right message to deliver. Please, make it less aggressive and prominent